include/cassandra.h

```diff
@@ -913,6 +914,58 @@ CASS_EXPORT void
 cass_cluster_set_token_aware_routing(CassCluster* cluster,
                                      cass_bool_t enabled);
 
+
+/**
+ * Configures the cluster to use latency-aware request routing, or not.
+ *
+ * Default is cass_true (enabled).
+ *
+ * This routing policy is a top-level routing policy. It uses the
+ * base routing policy to determine locality (dc-aware) and/or
+ * placement (token-aware) before considering the latency.
+ *
+ * @public @memberof CassCluster
+ *
+ * @param[in] cluster
+ * @param[in] enabled
+ */
+CASS_EXPORT void
+cass_cluster_set_latency_aware_routing(CassCluster* cluster,
+                                       cass_bool_t enabled);
+
+/**
+ * Configures the settings for latency-aware request routing.
+ *
+ * Defaults:
+ *
+ * exclusion_threshold: 2.0
+ * static_ms: 100 milliseconds
+ * retry_period_ms: 10,000 milliseconds (10 seconds)
+ * update_rate_ms: 100 milliseconds
+ * min_measured: 50
+ *
+ * @public @memberof CassCluster
+ *
+ * @param[in] cluster
+ * @param[in] exclusion_threshold Controls how much worse the latency must be compared to the
+ * average latency of the best performing node before it penalized.
+ * @param[in] scale_ms Controls the weight given to older latencies when calculating the average
```

Comment:
To my way of thinking, a 'scale' would not have units. However, I failed to understand the EMA math (comment below), so I'm not sure what to think about this.